js/settings.js

```diff
@@ -231,6 +231,8 @@ module.exports = function(opts) {
     }
     // basal rates
     else {
+      var activeSchedule = container.currentSettings().activeSchedule;
+      var displayName = datatype === activeSchedule ? datatype + ' (Active)' : datatype;
```

Comment:
Is this a common way we are setting variables? If it is then at least its consistent if its not then I feel that while succinct it reads a bit odd